### PR TITLE
Treewide: Add nicer names to modelfiles

### PR DESCRIPTION
**Pull request by FFHener**
This is related to #749

### CODE REVIEW

File: group_vars/model_ubnt_edgerouter_x_sfp.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 target: ramips/mt7621
+brand_nice: Ubiquiti
+model_nice: EdgeRouter x-SFP
```
Comment (Noki) [inline group_vars/model_ubnt_edgerouter_x_sfp.yml:4]: should be EdgeRouter X SFP

File: group_vars/model_ubnt_nanostation_ac_loco.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 target: ath79/generic
 openwrt_version: 22.03-SNAPSHOT
+brand_nice: Ubiquiti
+model_nice: Nanostation AC loco
```
Comment (Noki) [inline group_vars/model_ubnt_nanostation_ac_loco.yml:5]: should be "Loco"

File: group_vars/model_x86_64.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
-
 target: x86/64
 override_target: generic
 image_search_pattern: "*-ext4-combined.img*"
+model_nice: Virtual Machine
```
Comment (Noki) [inline group_vars/model_x86_64.yml:5]: this can also be bare metal

**Review by Noki — CHANGES_REQUESTED** [review]

File: group_vars/model_x86_64.yml (hunk 3 above)
Comment (FFHener) [reply group_vars/model_x86_64.yml]: What do you think about the new name?

**Review by Noki — APPROVED** [review]

**Pull request merged**